app/controllers/cards_controller.rb

```diff
@@ -16,18 +18,19 @@ def edit
   end
 
   def create
-    @card = Card.new(card_params)
+    @card = current_user.cards.build(card_params)
 
     if @card.save
-      redirect_to cards_path
+      redirect_to cards_path, notice: 'Карточка создана'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.